<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XXPlayMainWindowClass</class>
 <widget class="QWidget" name="XXPlayMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XXPlayMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="VideoWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>510</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:!hover{border-image: url(:/XXPlayMainWindow/Resources/open_normal.png);}
QPushButton:hover{border-image: url(:/XXPlayMainWindow/Resources/open_hot.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>510</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:!hover{border-image: url(:/XXPlayMainWindow/Resources/play_normal.png);}
QPushButton:hover{border-image: url(:/XXPlayMainWindow/Resources/play_hot.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="totaltime">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;000:00&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="sp">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="playtime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;000:00&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="XXSlider" name="playslider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>751</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">VideoWidget.h</header>
  </customwidget>
  <customwidget>
   <class>XXSlider</class>
   <extends>QSlider</extends>
   <header location="global">xxslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="XXPlayMainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>XXPlayMainWindowClass</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>874</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playslider</sender>
   <signal>sliderPressed()</signal>
   <receiver>XXPlayMainWindowClass</receiver>
   <slot>sliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>908</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playslider</sender>
   <signal>sliderReleased()</signal>
   <receiver>XXPlayMainWindowClass</receiver>
   <slot>sliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>898</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>XXPlayMainWindowClass</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>1053</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open()</slot>
  <slot>sliderPress()</slot>
  <slot>sliderRelease()</slot>
  <slot>play()</slot>
 </slots>
</ui>
